lxc/config_trust.go

```diff
@@ -139,7 +139,14 @@ func (c *cmdConfigTrustAdd) Run(cmd *cobra.Command, args []string) error {
 
 	cert := api.CertificatesPost{}
 
-	if len(args) == 0 {
+	// Check if remote is the first argument
+	// to detect method of adding trusted client
+	useToken := false
+	if len(args) == 0 || (len(args) == 1 && strings.Contains(args[0], ":")) {
```

Comment:
I think a better check here may actually be something like:
```
if len(args) == 0 || (len(args) == 1 && resource.name == "") {
```

That would be slightly cleaner than doing string matching for the remote separator.

Reply:
Makes sense. Thanks! Done.